extension/clear-purpose.html

```diff
@@ -11,25 +11,33 @@
+                    Extraneous information that is not directly relevant to the main purpose of a page is distinctly separated and programmatically determinable.
+                    For multi step tasks, signposts should be provided to clarify the broader context including steps completed, current step and steps pending.</strong></p>
 		        <h2><a id="user-content-suggestion-for-priority-level-aaaaaa" href="https://github.com/w3c/wcag21/wiki/Proposals-for-new-Success-Criteria#suggestion-for-priority-level-aaaaaa" aria-hidden="true"></a>Suggestion for Priority Level: (A)</h2>
 		        <h2><a id="user-content-related-glossary-additions-or-changes" href="https://github.com/w3c/wcag21/wiki/Proposals-for-new-Success-Criteria#related-glossary-additions-or-changes" aria-hidden="true"></a>Related Glossary additions or changes</h2>
 		        <p>None</p>
 		        <h2><a id="user-content-what-principle-and-guideline-the-sc-falls-within" href="https://github.com/w3c/wcag21/wiki/Proposals-for-new-Success-Criteria#what-principle-and-guideline-the-sc-falls-within" aria-hidden="true"></a>What Principle and Guideline the SC falls within.</h2>
 		        <p>Principle 3, Guideline 3.1 'Readable'</p>
 		        <h2><a id="user-content-description" href="https://github.com/w3c/wcag21/wiki/Proposals-for-new-Success-Criteria#description" aria-hidden="true"></a>Description</h2>
-		        <p>Each page or section of content should be organised so that it's purpose is obvious. This might be through the use of headings or a pyramid style of writing or even non textual markers (with suitable alternatives).
+		        <p>Each page or section of content should be organised and marked so  that it's purpose is obvious. This might be through the use of headings or labels or even a pyramid style of writing or even non textual markers (with suitable alternatives).
                     A clear organisation of content into pages and sections with obvious purpose alows users to more easily locate relevent sections and to be confident that those they read in detail will match their purposes. 
-                    There will also clearly be no need to read all content in case somehting important is missed.
-                    For example, Ads that appear in-line in a section of content are rarely related to the purpose of a section. They can be placed in a separate clearly delimited section</p>
+                    There will also be no need to read all content in case something important is missed. 
+                    For example, Ads that appear in-line in a section of content are rarely related to the purpose of a section and can be placed in a separate clearly delimited section</p>
+                    Here are some examples of how this will help people:</p> 
+                    <ul>
+                        <li>someone with dementia loses focus and then can not remember what they were doing</li>
```

Comment:
I added these examples here as per the template but now see @lseeman's comment to the group suggested add to Benefits section. Will move if really felt necessary